### PR TITLE
Add resolving tables and tables.reservations + fix menus

### PR DESCRIPTION
**Pull request by kevinschoonover**

### CODE REVIEW

File: server/src/controllers/Restaurant.ts [hunk 1]
```diff
@@ -45,7 +45,16 @@ export class RestaurantController {
   })
   @ResponseSchema(Restaurant)
   public async get(@Param('id') id: string) {
-    return Restaurant.findOne({ id });
+    return Restaurant.findOne(
```
Comment (cmparsons) [inline server/src/controllers/Restaurant.ts]: Just to clarify. The client only has to make one request to this endpoint, and it'll be given back a nested response with all information from sections, tables, and reservations.

Is there anything that is a part of the entities that is not returned in the response?

Comment (kevinschoonover) [reply server/src/controllers/Restaurant.ts]: Correct, it'll look roughly like
```
{
  sections: [
    {
      tables: []
    },
   {
      tables: []
    },
  ]
}
```

The things that will **not** be included are any other foreign keys such as `section.shifts` or `table.services`

Comment (cmparsons) [reply server/src/controllers/Restaurant.ts]: 👍

**Comment by cmparsons** [comment]
The tables and reservations stuff looks good to me. I would have someone else review the other changes before merging.

**Pull request merged**